languages/csharp/CommandRunner.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Bit.Sdk;
+
+internal class CommandRunner
+{
+
+    private readonly IntPtr _client;
+
+    internal CommandRunner(IntPtr client)
+    {
+        _client = client;
+    }
+
+    internal TReturn? RunCommand<TReturn>(Command command, Func<string, TReturn> deserializer)
+    {
+        var req = command.ToJson();
+        var result = BitwardenLibrary.RunCommand(req, _client);
+        return deserializer(result);
+    }
```

Comment:
Having to pass in a json serializer seems a bit cumbersome. What do you think about something like

```suggestion
    internal TReturn? RunCommand<TReturn>(Command command)
    {
        var req = command.ToJson();
        var result = BitwardenLibrary.RunCommand(req, _client);
        return JsonSerializer.Deserialize<TReturn>(result);
    }
```

(Note uses System.Text.Json)

Reply:
I just looked into this a bit more, seems the reason we did this in the first place was to prefer the generated `ToJson` and `FromJson`. I think we still want to do that, which means updating the consumers to use `ResponseForApiKeyLoginResponse.FromJson`.